libs/cardano-ledger-test/src/Test/Cardano/Ledger/Generic/Properties.hs

```diff
@@ -318,6 +325,25 @@ adaIsPreservedInEachEpoch proof genSize =
         trcInit = _traceInitState trc
         trcLast = lastState trc
 
+twiddleInvariantHolds ::
+  forall a era.
+  ( Arbitrary a,
+    Show a,
+    Twiddle a
+  ) =>
+  Proof era ->
+  TestTree
+twiddleInvariantHolds proof =
+  testProperty (show proof) $ twiddleInvariantProp @a
+
+twiddleInvariantHoldsEras :: TestTree
+twiddleInvariantHoldsEras =
+  testGroup
+    "Twiddle invariant holds for TxBody"
+    [ twiddleInvariantHolds @(AlonzoTxBody (AlonzoEra Mock)) (Alonzo Mock)
+    , twiddleInvariantHolds @(BabbageTxBody (BabbageEra Mock)) (Babbage Mock)
```

Comment:
You don't need actual `Proof` here, so just use an era name and also avoid using `Mock`, we should always whenever possible use `StandardCrypto`:

```suggestion
twiddleInvariantHolds ::
  forall a. (Arbitrary a, Show a, Twiddle a) =>
  String ->
  TestTree
twiddleInvariantHolds name =
  testProperty name $ twiddleInvariantProp @a

twiddleInvariantHoldsEras :: TestTree
twiddleInvariantHoldsEras =
  testGroup
    "Twiddle invariant holds for TxBody"
    [ twiddleInvariantHolds @(TxBody Alonzo) "Alonzo"
    , twiddleInvariantHolds @(TxBody Babbage) "Babbage"
```